test/sensitivities/linalg/blas.jl

```diff
@@ -0,0 +1,222 @@
+@testset "sensitivities/blas" begin
+
+let ϵ_abs = 1e-5, ϵ_rel = 1e-4, δ = 1e-6
```

Comment:
Are these values tight enough? Given a "tolerance factor" of `1e6`, I would require both absolute and relative error less than `1e-10`, assuming that intermediate computations are O(1).